content-security-policy/script-src/script-src-strict_dynamic_double_policy_report_only.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE HTML>
+<html>
+
+<head>
+    <title>A separate Report-Only policy does not influence `strict-dynamic` in the script-src directive.</title>
+    <script src='/resources/testharness.js' nonce='dummy'></script>
+    <script src='/resources/testharnessreport.js' nonce='dummy'></script>
+
+    <!-- CSP served:
+    1) Content-Security-Policy: script-src 'strict-dynamic' 'nonce-dummy'
+    2) Content-Security-Policy-Report-Only: script-src 'none'
+    -->
+</head>
+
+<body>
+    <h1>A separate Report-Only policy does not influence `strict-dynamic` in the script-src directive.</h1>
+    <div id='log'></div>
+
+    <script nonce='dummy'>
+        async_test(function(t) {
+            var handler = function(e) {
+                assert_equals(e.violatedDirective.split(' ')[0], 'script-src');
+                // Check that every violation comes from the Report-Only policy.
+                assert_equals(e.originalPolicy.trim(), "script-src 'none'");
```

Comment:
Yes, I do.

In `script-src-strict_dynamic_double_policy_honor_whitelist.sub.html`, I get:

`Uncaught Error: assert_equals: expected "script-src 'self' 'nonce-dummy'" but got " script-src 'self' 'nonce-dummy'"`

I don't know if it is due to the double policy, I am serving:

```
Content-Security-Policy: script-src 'strict-dynamic' 'nonce-dummy'
Content-Security-Policy: script-src 'self' 'nonce-dummy'
```

Is it a Chromium (Canary) bug? Shall I assert without .trim()?